plugins/examples/discovery-node-role-extension/src/test/resources/rest-api-spec/test/discovery_node_role_extension/10_basic.yml

```diff
@@ -0,0 +1,15 @@
+"additional role":
+    - skip:
+        reason: "contains"
+        features: contains
+
+    - do:
+        cluster.state: {}
+
+    # get master node id
+    - set: { master_node: master }
+
+    - do:
+        nodes.info: {}
+
+    - match: { nodes.$master.roles.0: "additional" }
```

Comment:
This requires #54376 so that we can reliably have it that the first item in the list is `additional`.

Reply:
This is integrated now.